ci/jenkins/jobs/macros.yaml

```diff
@@ -0,0 +1,11 @@
+- builder:
+    name: deploy-vm-and-run-tests-for-nephe
+    builders:
+            - shell: |-
+                    #!/bin/bash
+                    set -ex
+                    chmod +x ci/jenkins/nephe.sh
+                    ./ci/jenkins/nephe.sh --buildnumber "${{BUILD_NUMBER}}" --vchost "${{VC_HOST}}" \
+                            --vcuser "${{VC_USER}}" --datacenter "${{DATACENTERNAME}}" --datastore "${{DATA_STORE}}" \
+                            --vcCluster "${{VC_CLUSTER}}" --resourcePool "${{RESOURCEPOOLPATH}}" --vcNetwork "${{VMC_NETWORK_1}}" \
+                            --virtualMachine "${{VIRTUAL_MACHINE}}" --goVcPassword "${{GOVC_PASSWORD}}" --testUserName "${{TEST_USERNAME}}"
```

Comment:
Clean this up. Not required. Also, I am not sure what you passing arguments like this. the nephe.sh script has access to VC_USER anyway.. so directly access them, instead of passing like this